packages/calypso-e2e/src/lib/pages/plans-page.ts

```diff
@@ -51,8 +58,8 @@ export class PlansPage {
 	 * @throws If the expected plan title is not found in the timeout period.
 	 */
 	async validateActivePlanInMyPlanTab( expectedPlan: Plan ): Promise< void > {
-		await this.waitUntilLoaded();
-		await this.page.waitForSelector( selectors.myPlanTitle( expectedPlan ) );
+		const expectedPlanLocator = this.page.locator( selectors.myPlanTitle( expectedPlan ) );
+		await expectedPlanLocator.waitFor();
```

Comment:
This is not strictly related to this change, but I touched code around here and so decided to apply up to date approach of using Locators here.